apis/api-journeys/src/workers/migrations/button-blocks/add-submit-buttons.ts

```
@@ -0,0 +1,104 @@
+import { v4 as uuidv4 } from 'uuid'
+
+import { PrismaClient } from '.prisma/api-journeys-client'
```

ESLint

[error] Useless path segments for ".prisma/api-journeys-client", should be "./.prisma/api-journeys-client"

(import/no-useless-path-segments)